src/module-vsbridge-indexer-catalog/etc/mview.xml

```diff
@@ -12,7 +12,7 @@
             <table name="catalog_product_entity_varchar" entity_column="entity_id" />
             <table name="catalog_product_index_price" entity_column="entity_id"/>
             <table name="catalog_product_website" entity_column="product_id" />
-            <table name="catalog_category_product" entity_column="product_id" />
```

Comment:
don't delete catalog_category_product because if updates product into category table not calling execute current indexer

Reply:
thank you